README.md

```diff
@@ -3,101 +3,91 @@
 This is a [Home Assistant](https://www.home-assistant.io/) integration for the Israeli [cp.city-mind.com](https://cp.city-mind.com)
```

Comment:
Do we need to update here the url here from the old https://cp.city-mind.com to the new https://rym-pro.com/ website?